src/components/Launcher.js

```diff
@@ -21,14 +30,21 @@ class Launcher extends Component {
     });
   }
 
+  //call this function to open chat window
+  openChat() {
+    this.setState({
+      active: true
+    });
+  }
+
   render() {
     const classList = [
       'sc-launcher',
       (this.state.active ? ' active' : ''),
     ];
     return (
       <div>
-        <div className={classList.join(' ')} onClick={this.handleClick.bind(this)}>
+        <div className={classList.join(' ')} onClick={this.handleClick}>
```

Comment:
Why remove the bind?

Reply:
I've been told that best practice is to bind the functions in the constructor, as render gets called often?

Reply:
.bind also returns a new function, so you create a new function multiple times

Reply:
Ah yea, that's definitely true. I don't think that that is going to be a super big deal here given how infrequently this is going to be called and how small the f's are. But you're right.

From a personal style point of view, I don't really like binding in the constructor so I tend to just not.

That said, if you want to go through and rm all the binds in all the components and move them all to the constructors to keep the style consistent, I would merge that as a seperate PR.